screens/Scan/ScanScreen.tsx

```diff
@@ -151,18 +151,44 @@ export const ScanScreen: React.FC = () => {
   function displayStorageLimitReachedError(): React.ReactNode {
     return (
       !controller.isEmpty && (
-        <ErrorMessageOverlay
+        <MessageOverlay
           isVisible={
             controller.isMinimumStorageRequiredForAuditEntryLimitReached
           }
           translationPath={'ScanScreen'}
           error="errors.storageLimitReached"
-          onDismiss={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
+          onBackdropPress={() => navigation.navigate(BOTTOM_TAB_ROUTES.home)}
         />
       )
     );
   }
 
+  function displayInvalidQR(): React.ReactNode {
```

Comment:
Can we rename this method name? This looks like we are displaying invalid QR 
cc: @sreenadh

Reply:
changed it to `displayInvalidQRpopup`